app/src/main/java/com/vero/tasky/agenda/data/local/dao/TaskDao.kt

```diff
@@ -22,6 +22,12 @@ interface TaskDao {
     """)
     fun loadTasksForDay(from: Long, to: Long) : Flow<List<TaskEntity>>
 
+    @Query("SELECT * FROM task WHERE id = :id")
+    fun loadTaskFlow(id: String) : Flow<TaskEntity?>
```

Comment:
When you are just getting 1 element, 99% of the times its not necessary to have a flow
Since you wont be listening to changes